src/query/execute_schema.c

```diff
@@ -88,6 +88,13 @@
 #define MAX_FILTER_PREDICATE_STRING_LENGTH (1073741823)
 #define MAX_FUNCTION_EXPRESSION_STRING_LENGTH 1024
 
+namespace hnsw
+{
+  int m;
+  int ef_construction;
+  enum DB_VECTOR_DISTANCE_METRIC metric;
+};
```

Comment:
```suggestion
namespace hnsw
{
  PT_VECTOR_INDEX_INFO vindex_info;
};
```
PT_VECTOR_INDEX_INFO 구조와 완전히 동일한 것 같은데, 이 구조를 그대로 재사용해도 괜찮지 않을까요